clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -39,11 +39,19 @@ import (
 const (
 	apiResponsivenessPrometheusMeasurementName = "APIResponsivenessPrometheus"
 
+	// TODO(krzysied): figure out why we're getting non-capitalized proxy and fix this
```

Comment:
I'm almost sure it was because of https://github.com/kubernetes/kubernetes/issues/81659